src/docfx/lib/PathUtil.cs

```diff
@@ -0,0 +1,133 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Text;
+
+namespace Microsoft.Docs
+{
+    /// <summary>
+    /// Provide utils of path
+    /// </summary>
+    public static class PathUtil
+    {
+        /// <summary>
+        /// Determines if a file or path is inside a folder. Input should both be normalized.
+        /// </summary>
+        public static bool Inside(string folder, string path)
+        {
+            Debug.Assert(folder.Length > 0 && folder[folder.Length - 1] == '/');
+
+            return folder == "./" || path.StartsWith(folder, StringComparison.OrdinalIgnoreCase);
+        }
+
+        /// <summary>
+        /// Create a relative path from one path to another file.
+        /// Use this over <see cref="Path.GetRelativePath(string, string)"/> when
+        /// <paramref name="fileRelativeTo"/> is a file.
```

Comment:
fix this

Reply:
it's a ref in the summary

---
In reply to: [180008851](https://github.com/dotnet/docfx/pull/2601#discussion_r180008851) [](ancestors = 180008851)